bot/main.py

```diff
@@ -235,27 +264,45 @@ async def backend_info(ctx: commands.Context):
     )
 
 
-def is_academic(ctx: commands.Context):
-    """Checks if the server is an academic server."""
+@backend_info.error
+async def backend_info_error(ctx: commands.Context, error: Exception):
+    """If the author of the message is not a bot admin then reply accordingly."""
+    if (
+        isinstance(error, CheckFailedException)
+        and error.check_name == "is_author_bot_admin"
+    ):
+        author = ctx.message.author
+        await ctx.reply(f"{author.mention} is not a bot admin.", ephemeral=True)
+    else:
+        await ctx.reply("Some checks failed.", ephemeral=True)
+
+
+def is_academic_or_bot_admin(ctx: commands.Context):
+    """Checks if the server is an academic server or if the author is a bot admin."""
     if ctx.guild is None:
         return False
 
     try:
-        return server_configs[ctx.guild.id]["is_academic"]
-    except KeyError:
-        return False
+        author = ctx.message.author
+        is_admin = is_bot_admin(author.id)
+        if not server_configs[ctx.guild.id]["is_academic"] and not is_admin:
+            raise CheckFailedException("is_academic_or_bot_admin")
+        return True
+    except KeyError as err:
+        raise CheckFailedException("server_config") from err
```

Comment:
It would be better to leave this function as it is, as `is_academic`, and instead define a new function 

```py
def is_academic_or_bot_admin(ctx: commands.Context):
    """Checks if the server is an academic server or if the author is a bot admin."""
    return is_author_bot_admin(ctx) or is_academic(ctx)
```

Of course, you would need to modify `is_academic` to raise `CheckAcademic` errors instead of returning `False`

Reply:
I just realised that `@commands.check_any` exists. We can make the implementation much simpler with this!